client/containers/DashboardActivity/DashboardActivity.tsx

```diff
@@ -43,7 +46,30 @@ const DashboardActivity = (props: Props) => {
 	});
 
 	return (
-		<DashboardFrame className="dashboard-activity-container" title="Activity">
+		<DashboardFrame
+			className="dashboard-activity-container"
+			title="Activity"
+			controls={
+				canEnrollInActivityDigestEmails && (
+					<Switch
+						checked={member.receivesDigestEmail}
+						className="parent-approval-switch"
+						onChange={(e) =>
```

Comment:
Would you mind factoring out a local `setEnrolledInActivityDigest(enrolled: boolean) => Promise<Partial<Member>>` here? The inlining is visually busy.